_includes/hero.html

```diff
@@ -1,6 +1,6 @@
 <section class="hero">
   <div class="logo">
-    <img min-height="60px" alt="" src="/static/img/libera-color.svg">
+    <img min-height="60px" alt="Libera Logo" src="/static/img/libera-color.svg">
```

Comment:
This purposefully has an empty alt="" to mark is as decorative, since the text "Libera Chat coems after

Reply:
htmlproofer complains as it was... can disable.

Reply:
@swantzter I'm not clear why it being decorative implies it shouldn't have an alt tag. I feel it should contain this useful information for users if the image cannot be displayed?

Reply:
I'd say this falls under "image with adjacent text alternative" in the WAI guidelines https://www.w3.org/WAI/tutorials/images/decorative/#image-with-adjacent-text-alternative

Basically having an alt text here means text-only browsers, or screen readers will read "Libera.Chat, Libera.Chat"

Reply:
Fair enough, will get rule disabled there.